RUFAS/EEE/emissions.py

```diff
@@ -283,387 +205,438 @@ def _get_feed_emissions_data(
-                        f",layer=.*",
-                    ],
-                    "slice_start": SLICE_START,
-                },
-                "nitrous_oxide": {
-                    "name": "Soil Nitrous Oxide emissions",
-                    "description": "Collects the nitrous oxide emissions from all soil layers in the field in the "
-                    "last year of the simulation.",
-                    "filters": [
-                        f"FieldDataReporter.send_daily_variables.nitrous_oxide_emissions.field='{sanitized_name}',"
-                        f"layer=.*"
-                    ],
-                    "slice_start": SLICE_START,
-                },
-            }
-            soil_data = self._soil_data_update(filters)
-
-            starting_carbon_stock_filter = {
-                "name": "Starting soil profile carbon stock",
-                "description": "Collects the soil carbon stock level 365 days before the simulation ended.",
-                "filters": [
-                    f"FieldDataReporter.send_daily_variables.total_soil_carbon_amount.field='{sanitized_name}'"
-                ],
-                "slice_start": SLICE_START,
-                "slice_end": SLICE_END,
-            }
+        )
 
-            starting_carbon_stock = self.om.filter_variables_pool(starting_carbon_stock_filter)
+        self._report_daily_farmgrown_feed_fed_emissions_and_resources(
+            daily_farmgrown_feed_fed_emissions_and_resources_by_feed_id
+        )
 
-            total_starting_carbon = sum(
-                [starting_carbon_stock[key]["values"][0] for key in starting_carbon_stock.keys()]
-            )
+        farm_grown_feeds_fed_to_animals = list(daily_farmgrown_feed_fed_emissions_and_resources_by_feed_id.keys())
+        self._calculate_and_report_lca_and_luc_emissions(farm_grown_feeds_fed_to_animals, feed_deductions_data)
 
-            ending_carbon_stock_filter = {
-                "name": "Ending soil profile carbon stock",
-                "description": "Collects the soil carbon stock level on the last day of the simulation.",
-                "filters": [
-                    f"FieldDataReporter.send_daily_variables.total_soil_carbon_amount.field='{sanitized_name}'"
-                ],
-                "slice_start": FINAL_DAY_SLICE_START,
-            }
-            ending_carbon_stock = self.om.filter_variables_pool(ending_carbon_stock_filter)
-            total_ending_carbon = sum([ending_carbon_stock[key]["values"][0] for key in ending_carbon_stock.keys()])
+    def _parse_farmgrown_feeds_emission_data(self) -> dict[str, dict[str, dict[int, float]]]:
+        """
+        Parses farmgrown feeds emission data from the OutputManager and returns a dictionary with emission data for
+        each field on every simulation day.
+        """
+        result: dict[str, dict[str, dict[int, float]]] = defaultdict(dict)
```

Comment:
```suggestion
        application_data: dict[str, dict[str, dict[int, float]]] = defaultdict(dict)
```

or `field_application_data` - something more informative than `result`